samples/tutorials/Data passing in python components.ipynb

```diff
@@ -5,7 +5,7 @@
    "metadata": {},
```

Comment:
<p>typo: Smal -&gt; Small</p><p> </p><p>Also, resonating on Amy's question: do we have a concrete understanding on the data size that can be passed as raw, vs. can only be passed by reference?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/pipelines/pull/2868/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Fixed the typo.